draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -320,66 +464,90 @@ Internet-Draft      Network Slice Topology Data Model       January 2024
-   may be further supported by an end-to-end tunnel that traverse a
-   specific path in a given customized topology supplied by the
-   customer.  The resources associated with a link are immediately
-   commissioned by the realization at the time of network slice
-   configuration.
-
-   Alternatively, a customer can request resources to be reserved for
-   potential network slices through a customized topology, and use the
-   resources to build network slices in the future.  The customized
-   topology represents resources that are reserved but not commissioned
-   at the time of request.  By doing so, customers can share resources
-   between multiple endpoints and use them on demand.
-
-   In the example shown in Figure 2, two customized topology intents
-   named as Network Slice Blue and Network Slice Red, are created by
-   separate customers and delivered to the network slice service
-   provider.  The provider maps the two intents to corresponding network
-   resource partitions (NRPs) internally.  In realizing the network
-   resource partitions, node virtualization is used to separate and
-   allocate resources in physical devices.  Two virtual routers VR1 and
-   VR2 are created over physical router R1, and two virtual routers VR3
-   and VR4 are created over physical router R2, respectively.  Each of
-   the virtual routers,as a partition of the physical router, takes a
+Internet-Draft      Network Slice Topology Data Model      February 2024
+
+
+   In the example shown in Figure 2, two topology intents named as
+   Network Slice Blue and Network Slice Red, are created by separate
+   customers and delivered to the network slice service provider.  The
+   provider maps the two intents to corresponding network resource
+   partitions (NRPs) internally.  In realizing the network resource
+   partitions, node virtualization is used to separate and allocate
+   resources in physical devices.  Two virtual routers VR1 and VR2 are
+   created over physical router R1, and two virtual routers VR3 and VR4
+   are created over physical router R2, respectively.  Each of the
+   virtual routers,as a partition of the physical router, takes a
    portion of the resources such as ports and memory in the physical
    router.
    Depending on the requirements and the implementations, they may share
    certain resources such as processors, ASICs, and switch fabric.
 
+   A network slice customer has the capability to configure customer
+   intent topologies without needing any prior knowledge of the
+   provider's network or resource availability.  However, this approach
+   could potentially create challenges for the provider in understanding
+   and realizing the intended topology.
+
+   Alternatively, the provider can choose to describe the available
+   resources and capabilities in the form of an abstract topology, which
+   is then exposed to the customer before network slice requests.  By
```

Comment:
We need to be clear here what this abstract topology should be: I mean, before we said that "a minimum level of information for the provider is needed for the customer to make a request for an intent topology. Now we are talking about abstract topo exposed by provider. This surely makes sense but we need to clarify which level this abstract topology is. Is the same level of information we have talked about before in the I-D? E.G. SAP topology ?

Reply:
I think the minimum level of information referred before is required for network slicing, while the abstract topology referred here is totally optional and up to the provider.